app/parsers/hyku_addons/csv_parser.rb

```diff
@@ -1,6 +1,12 @@
 # frozen_string_literal: true
 module HykuAddons
   class CsvParser < Bulkrax::CsvParser
+    # FIXME: Override to make debugging easier
+    def perform_method
+      return :perform_now if Rails.env == 'development' || Rails.env == 'test'
```

Comment:
maybe I would replace that if for:

```suggestion
      return :perform_now unless Rails.env.production?
```

WDYT?

Reply:
Yes, I forgot about that.  I'll get that in the next PR.